source/transactions-convenient-api/tests/README.md

```diff
@@ -41,8 +41,37 @@ If possible, drivers should implement these tests without requiring the test run
 the retry timeout. This might be done by internally modifying the timeout value used by `withTransaction` with some
 private API or using a mock timer.
 
+### Retry Backoff is Enforced
+
+Drivers should test that retries within `withTransaction` do not occur immediately. First, run transactions without
+backoff. To do so, configure the random number generator used for jitter to always return `0` -- this effectively
+disables backoff. Then, configure a fail point that forces 30 retries like so:
+
+```json
+{
+    "configureFailPoint": "failCommand",
+    "mode": {
+        "times": 13
+    },
+    "data": {
+        "failCommands": ["commitTransaction"],
+        "errorCode": 251,  // NoSuchTransaction
+    },
+}
+```
+
+Let the callback for the transaction be a simple `insertOne` command. Let `no_backoff_time` be the time it took for the
+command to succeed.
+
+Next, we will run the transactions again with backoff. Configure the random number generator used for jitter to always
+return `1`. Set the fail point to force 13 retries using the same command as before. Using the same callback as before,
+check that the total time for the withTransaction command is within +/-1 second of `no_backoff_time` plus 2.2 seconds.
+Note that 2.2 seconds is the sum of backoff 13 consecutive backoff values and the 1-second window is just to account for
+potential networking differences between the two runs.
```

Comment:
minor nit for tone:

"the 1-second window is just to account for
potential networking differences between the two runs."
->
"the 1-second window accounts for potential variance between the two runs.

Reply:
changed in [56e88f0](https://github.com/mongodb/specifications/pull/1851/commits/56e88f080ba5c8461bad92cd206ac47360957cbe)